openkaito/utils/config.py

```diff
@@ -203,6 +203,13 @@ def add_args(cls, parser):
             default=False,
         )
 
+        parser.add_argument(
+            "--blacklist.validator_min_stake",
+            type=int,
+            help="If set, miners will blacklist requests from validators with stake less than this value.",
+            default=10000,
```

Comment:
According to https://docs.bittensor.com/whats-new-in-docs#11-march-2024 , the bittensor min limit is 1000, can we set the default to 1000 and let the miners customize if they want it to be higher?